samples/corda/corda-simple-application/clients/src/main/resources/config/credentials/remote-network-config.json.template

```diff
@@ -0,0 +1,26 @@
+{
+  "network1": {
+    "type": "fabric",
+    "relayEndpoint": "localhost:9080",
+    "channelName": "mychannel",
+    "chaincode": "simpleassettransfer"
+  },
+  "network2": {
+    "type": "fabric",
+    "relayEndpoint": "localhost:9083",
+    "channelName": "mychannel",
+    "chaincode": "simpleassettransfer"
+  },
+  "Corda_Network": {
+    "type": "corda",
+    "relayEndpoint": "localhost:9081",
+    "partyEndPoint": "localhost:30006",
```

Comment:
Party Endpoint for Corda_Network is `localhost:10006` not 30006. Also fix the one in Corda_Network2 below, that should be 30006 port.

Reply:
Done.